besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -1839,11 +1840,21 @@ public void validateRpcOptionsParams() {
                 || rpcEndpointServiceImpl.hasNamespace(apiName);
 
     if (!jsonRPCHttpOptionGroup.rpcHttpApis.stream().allMatch(configuredApis)) {
-      throw new ParameterException(this.commandLine, "Invalid value for option '--rpc-http-apis'");
+      List<String> invalidHttpApis = new ArrayList<String>(jsonRPCHttpOptionGroup.rpcHttpApis);
+      invalidHttpApis.removeAll(AVAILABLE_APIS);
```

Comment:
it might be clearer to have this on one line
invalidHttpApis = new ArrayList<String>(jsonRPCHttpOptionGroup.rpcHttpApis).removeAll(AVAILABLE_APIS)